v2/mfa/backend/first-factor.mdx

```diff
@@ -385,8 +385,52 @@ Session.init({
 TODO
 </TabItem>
 <TabItem value="python">
-TODO
+
+```python
+from supertokens_python import init, InputAppInfo, SupertokensConfig
+from supertokens_python.recipe import session
+from supertokens_python.recipe.session.interfaces import RecipeInterface
+from typing import Any, Dict, Union
+
+# highlight-start
+def override_session_functions(original_implementation: RecipeInterface):
+    original_create_new_session = original_implementation.create_new_session
+
+    async def create_new_session(
+        request: Any,
+        user_id: str,
+        access_token_payload: Union[None, Dict[str, Any]],
+        session_data: Union[None, Dict[str, Any]],
+        user_context: Dict[str, Any],
+    ):
+        # Insert "is2faComplete" in access token payload
+        if access_token_payload is None:
+            access_token_payload = {}
+
+        access_token_payload = {**access_token_payload, "is2faComplete": False}
+        return await original_create_new_session(
+            request, user_id, access_token_payload, session_data, user_context
+        )
+
+    original_implementation.create_new_session = create_new_session
+    return original_implementation
+# highlight-end
+
+init(
+    supertokens_config=SupertokensConfig("http://localhost:3567"),
+    app_info=InputAppInfo(api_domain="...", app_name="...", website_domain="..."),
+    framework="flask",
+    recipe_list=[
+        session.init(
+            # highlight-next-line
+            override=session.InputOverrideConfig(functions=override_session_functions)
+        )
+    ],
+)
```

Comment:
You can just do session.init and not give the whole init

Reply:
done.